src/views/LoginView.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
+    class="w-full min-h-screen bg-gray-50 flex flex-col sm:justify-center items-center pt-6 sm:pt-0"
+  >
+    <div class="w-full sm:max-w-md p-5 mx-auto">
+      <h2 class="mb-12 text-center text-5xl font-extrabold">Welcome.</h2>
+      <form>
+        <div class="mb-4">
+          <label class="block text-md font-light mb-2" for="username"
+            >Username</label
+          >
+          <input
+            v-model="inputUserName"
+            class="py-2 px-3 border border-gray-300 focus:border-red-300 focus:outline-none focus:ring focus:ring-red-200 focus:ring-opacity-50 rounded-md shadow-sm disabled:bg-gray-100 mt-1 block w-full"
+            type="text"
+            name="username"
+            id=""
+          />
+        </div>
+        <div class="mb-4">
+          <label class="block text-md font-light mb-2" for="password"
+            >Password</label
+          >
+          <input
+            v-model="inputPassword"
+            class="py-2 px-3 border border-gray-300 focus:border-red-300 focus:outline-none focus:ring focus:ring-red-200 focus:ring-opacity-50 rounded-md shadow-sm disabled:bg-gray-100 mt-1 block w-full"
+            type="password"
+            name="password"
+            id=""
+          />
+        </div>
+        <div class="pt-1 px-20">
+          <label
+            class="p-1 mb-5 text-sm text-red-700 bg-red-100 rounded-lg dark:bg-red-200 dark:text-red-800"
+            role="alert"
+            v-show="errorMessage"
+            >{{ message }}</label
+          >
+        </div>
+
+        <div class="flex items-center justify-between mb-5">
+          <button
+            @click="loginUser"
+            class="w-full inline-flex items-center justify-center px-4 py-2 bg-orange-500 hover:bg-orange-600 text-white font-light py-2 px-6 rounded focus:ring focus:ring-offset-amber-400 disabled:opacity-25 transition"
+            type="button"
+          >
+            Sing In
+          </button>
```

Comment:
its really a small detail but in tailwind config file I added our custom colours so I think it would be great if we used them everywhere on the website to keep it consistent
<img width="407" alt="Screenshot 2022-05-07 at 11 17 41" src="https://user-images.githubusercontent.com/71661786/167247842-bea90382-ac1a-49b7-8ae5-adc3cc0a9f3c.png">
so instead of 'bg-orange-500' could you use 'bg-primary-orange'? of course if you like this colour more you can change the value in tailwind file so it will change everywhere :)